### ISSUE TITLE
Appending to a post's tags in tests

### ISSUE THREAD

**Issue by hasandiwan** [diaspora/diaspora#8328]
I'm attempting to add openpgp-signed StatusUpdates to diaspora, if there's a 'signed' tag:

`
> if @s.tag_list.member?('signed') then
>   pgp_sign @s
> end
> @s.save!
`

It's quite straightforward to code, but I'd now like to verify this functionality works. What I have so far is:

`
> @s.tag_list = ActsAsTaggableOn::TagList.new(:signed)
> @s.tag_list 
> @s.tag_list
 => ["signed"]
> @s.save!
> @s.reload
> @s.tag_list
 => [] 
 `

Huh?

**Comment by jhass** [comment]
The forum might be a little bit better avenue for such talk than the issue tracker :) 

`StatusMessage` includes `Diaspora::Taggable`

https://github.com/diaspora/diaspora/blob/37a7c0b35d560125fb0e60081f71fa029a5adfa4/app/models/status_message.rb#L8

Which defines a before validation hook
https://github.com/diaspora/diaspora/blob/37a7c0b35d560125fb0e60081f71fa029a5adfa4/lib/diaspora/taggable.rb#L25

that sets the tag list from what's parsed from the message body

https://github.com/diaspora/diaspora/blob/37a7c0b35d560125fb0e60081f71fa029a5adfa4/lib/diaspora/taggable.rb#L36-L45

In other words a message's tags are not meant to be a generic metadata attachment system, they're purely there to support the hashtagging feature.
